vortexasdk/endpoints/cargo_movements.py

```diff
@@ -36,8 +36,8 @@ def __init__(self):
     def search(
         self,
         filter_activity: str = None,
-        filter_time_min: datetime = datetime(2019, 10, 1, 0),
-        filter_time_max: datetime = datetime(2019, 10, 1, 1),
```

Comment:
Can we remove this entirely? The default is already set at the API level

Reply:
these fields are marked as required in our docs so I would keep it to be consistent